src/index.js

```diff
@@ -9,6 +9,10 @@ var SVG_NS = "http://www.w3.org/2000/svg"
 var EMPTY_OBJECT = {}
 var EMPTY_ARRAY = []
 
+var CREATE_EVENT = new Event('create')
```

Comment:
Depending on browser support, we may want to use `event = new CustomEvent('build', { detail: elem.dataset.time });`.

Another thing I think we need to consider is running in headless - will this still work as intended?

Reply:
From what I have read.. `Event` is supported better than `CustomEvent` but we might need to *not* use the constructor. Something like this (which is not ideal I guess but will work):

```
var event;
if(typeof(Event) === 'function') {
    event = new Event('submit');
}else{
    event = document.createEvent('Event');
    event.initEvent('submit', true, true);
}
```